src/components/Home/StepperSection.tsx

```diff
@@ -0,0 +1,356 @@
+        fontSize: '1.728rem',
+     
+      }}
+      component="b"
+    >
+      Let's discover what the internet knows about you
+    </Typography>,
+  ];
+}
+
+function getStepContent(step: number) {
+  switch (step) {
+    case 0:
+      return (
+        <VideoSection>
+          <Box
+            display="flex"
+            flexDirection="column"
+            justifyContent="center"
+            gap="1.125rem"
+          >
+            <Typography
+              sx={{
+                lineHeight: '2rem',
+                fontWeight: 400,
+                fontSize: '1.2rem',
+              }}
+            >
+              You can start by requesting your personal data from Google and
+              Facebook.{' '}
+            </Typography>
+            <Typography
+              sx={{
+                lineHeight: '2rem',
+                fontWeight: 400,
+                fontSize: '1.2rem',
+              }}
+            >
+              It can take up to 48h to receive your data files : Facebook will
+              send you a notification and Google an e-mail when your files are
+              ready to be downloaded.{' '}
+            </Typography>
+            <Typography
+              sx={{
+                lineHeight: '2rem',
+                fontWeight: 400,
+                fontSize: '1.2rem',
+              }}
+            >
+              At the end of this step, you should have received multiple files
+              in Zip format. Download them locally on your device.
+            </Typography>
+          </Box>
+          <Box width={'100%'}>
+            <video width="100%" height="auto" controls>
+              <source
+                src={require('../../assets/video/video-test.mp4')}
+                type="video/mp4"
+              />
+            </video>
+          </Box>
+        </VideoSection>
+      );
+    case 1:
+      return (
+        <DropZoneSection>
+          <Box
+            display="flex"
+            flexDirection="column"
+            justifyContent="center"
+            gap="0.813rem"
+          >
+            <Typography
+              sx={{
+                lineHeight: '2rem',
+                fontWeight: 400,
+                fontSize: '1.2rem',
+              }}
+            >
+              Once you’ve received your data, you can initialize your Avastar
+              digital identity card. Upload your data files right here and
+              finally access what you always wanted to know about your personal
+              data !
+            </Typography>
+            <Typography
+              sx={{
+                lineHeight: '2rem',
+                fontWeight: 400,
+                fontSize: '1.2rem',
+              }}
+            >
+              The data you upload will not reach any server. The files you
+              upload never leave your computer, all your data is stored locally
+              on your browser. It will also be deleted every time your session
+              is refreshed
```

Comment:
Ici, utiliser ce texte pour éviter d'utiliser upload : "We can’t see the data you collected from platforms as it’s stored and processed on your browser, so you are sure that your data never leaves your computer and doesn’t reach our servers."

Reply:
Super, c'est modifié :+1: